android/package.json

```diff
@@ -1,17 +1,18 @@
 {
-	"name": "sample",
-	"description": "Sample localization project",
-	"repository": "git@github.com:iLib-js/ilib-loctool-samples.git",
-	"license": "Apache-2.0",
-	"version": "1.0.0",
-	"scripts": {
-		"loc": "loctool",
-		"debug": "node --inspect-brk node_modules/loctool/loctool.js",
-		"clean": "rm -rf *.xliff res/values-* res/values/strings-auto.xml"
-	},
-	"devDependencies": {
-		"ilib-loctool-android-resource": "^1.0.1",
-		"ilib-loctool-android-layout": "^1.0.1",
-		"loctool": "^2.16.2"
-	}
+    "name": "sample",
+    "description": "Sample localization project",
+    "repository": "git@github.com:iLib-js/ilib-loctool-samples.git",
+    "license": "Apache-2.0",
+    "version": "1.0.0",
+    "scripts": {
+        "loc": "loctool",
+        "debug": "node --inspect-brk node_modules/loctool/loctool.js",
+        "clean": "rm -rf *.xliff res/values-* res/values/strings-auto.xml",
+        "test": "echo success"
```

Comment:
It means nothing. is there a reason to add it?

Reply:
npm-check-updates runs the "test" script to see if it broke anything before and after it upgrades the version numbers. It needs to be there in order to have it update automatically.

Reply:
Check it out: https://www.npmjs.com/package/npm-check-updates